workflow/cron/operator.go

```diff
@@ -113,9 +113,27 @@ func (woc *cronWfOperationCtx) persistUpdate() {
 		woc.log.WithError(err).Error("failed to marshall cron workflow status data")
 		return
 	}
-	err = wait.ExponentialBackoff(retry.DefaultBackoff, func() (bool, error) {
+
+	woc.persist(data)
+}
+
+func (woc *cronWfOperationCtx) persistUpdateActiveWorkflows() {
+	data, err := json.Marshal(map[string]interface{}{"status": map[string]interface{}{"active": woc.cronWf.Status.Active}})
+	if err != nil {
+		woc.log.WithError(err).Error("failed to marshall cron workflow status.active data")
+		return
+	}
+
+	woc.persist(data)
+}
+
+func (woc *cronWfOperationCtx) persist(data []byte) {
```

Comment:
call this patch? take a interfare{} as the argument and return the error?

Reply:
Sure I like this